lib/elixir/src/elixir_erl_for.erl

```diff
@@ -177,22 +177,11 @@ build_into(Ann, Clauses, Expr, Into, Uniq, S) ->
 
 %% TODO: Remove this check once we support Erlang/OTP 21+ exclusively.
```

Comment:
Please remove the TODO. :D

Reply:
I pushed the fix in the same time as you have been commenting ;)